src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcReadOnlyTest.java

```diff
@@ -36,27 +41,57 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
 
 /** This test class runs a SQL script for testing a connection in read-only mode. */
 @Category(ParallelIntegrationTest.class)
-@RunWith(JUnit4.class)
+@RunWith(Parameterized.class)
 public class ITJdbcReadOnlyTest extends ITAbstractJdbcTest {
   private static final long TEST_ROWS_COUNT = 1000L;
 
+  @Parameters(name = "Dialect = {0}")
+  public static List<DialectTestParameter> data() {
+    List<DialectTestParameter> params = new ArrayList<>();
+    final Map<String, String> googleStandardSqlScripts =
+        ImmutableMap.of("TEST_READ_ONLY", "ITReadOnlySpannerTest.sql");
+    final Map<String, String> postgresScripts =
+        ImmutableMap.of("TEST_READ_ONLY", "ITPgReadOnlySpannerTest.sql");
```

Comment:
regarding `ITPgReadOnlySpannerTest.sql` and `ITPg*.sql` files: Should we consider moving them to a separate folder with more verbose name of the folder?

Reply:
I think that's a good idea. I've moved the PG script files to a separate PostgreSQL folder.